src/modules/developer_tools.json

```diff
@@ -0,0 +1,250 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "DeveloperTools",
+        "description": "A module for supporting off-device developer tools.",
+        "version": "0.0.0"
+    },
+    "methods": [
+        {
+            "name": "find",
+            "summary": "Starts a long-running DNS-SD search for a list of developer tools that match the standard 'Firebolt Developer Tool' service type",
+            "tags": [],
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "const": null
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
+        },
+        {
+            "name": "stopFind",
+            "summary": "Stops the long-running find operation",
+            "tags": [],
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "const": null
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
+        },
+        {
+            "name": "pair",
```

Comment:
Just FYI, I don't have strong feelings on this at all.

Wifi calls this "connect" and "disconnect" but I think that's pretty accepted verbiage for WiFi, that we don't want to force on all similar APIs.

If we had a Bluetooth module we'd definitely use "pair"

Does DNS-SD have a standard way to say this?

Reply:
There's no real notion of "pair" in DNS-SD... Once you've discovered a service (gotten its host/port, or in our case an id that hides that info), you do whatever you want to do to it / with it. Send it printer commands, whatever. In our case, we want to "pair" with the dev tool and start accepting Firebolt JSON RPC messages and executing them. "Pair" seems like the right term here.